monitoring/v1alpha1/types.go

```diff
@@ -331,8 +331,8 @@ type RelabelConfig struct {
 	// action to perform based on regex matching. Must be one of: replace, keep,
 	// drop, hashmod, labelmap, labeldrop, or labelkeep.  Default is: 'replace'
```

Comment:
This comment should be updated to include the PascalCase versions instead of the lower case versions

Reply:
Thanks, I'll fix that.

Just for context, I was using https://github.com/prometheus-operator/prometheus-operator/blob/9bef66a611c01c60faacb705960b98a3fba7dd6d/pkg/apis/monitoring/v1/types.go#L1106 as inspiration.

Reply:
Unfortunately we don't control that upstream API. But within OpenShift, we are trying to keep the API consistent no matter the upstream or backend project, hence sticking with just PascalCase

Reply:
I added another commit to make the field names in the doc comments actually match the CRD field names. WDYT?

Reply:
We actually prefer for the names to match the json tag names, that way when a user does `oc explain` it shows the json tag name, which is what they will interact with in their YAML file. I would drop the second commit from this PR, but the first commit LGTM I think

Reply:
Thanks I'll drop that. Based on a few examples I looked at I wasn't sure what the preferred variant is.